.github/quarkus-github-bot.yml

```diff
@@ -285,13 +285,17 @@ triage:
         - extensions/resteasy-classic/resteasy-qute/
     - id: reactive-messaging
       labels: [area/reactive-messaging, area/smallrye]
+      expression: |
+        matches("reactive.messaging", title)
+        || matches("mqtt", title)
+        || matches("rabbitmq", title)
       title: "reactive.messaging"
```

Comment:
Does it work to have both or should I remove the `title` line?

Reply:
You can remove the `title` here, as you are already testing it in the expression